solutions/AC Matching.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: ac-matching
+source: AtCoder DP Contest
+title: Matching
+author: Neo Wang
+---
+
+## Explanation
+
+If we define $\texttt{dp}[S]$ to be the amount of matchings for females in set
+$S$, this problem boils down to the following formulation:
```

Comment:
Seems pretty difficult to understand. Shouldn't $dp[S]$ be the number of ways to match the females in $S$ to the first $|S|$ males?